Bottom: 0a99e4d62801afe34959432eb3fd0fca8e1595e8
Top:    ec0a7ef983b3c998ee277cc244e0bda3350b96c2
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-08-15 21:30:32 -0400

Refresh of pwm

---

diff --git a/sound/soc/generic/simple-card.c b/sound/soc/generic/simple-card.c
index 2bc4085..231d2c2 100644
--- a/sound/soc/generic/simple-card.c
+++ b/sound/soc/generic/simple-card.c
@@ -8,9 +8,6 @@
  * it under the terms of the GNU General Public License version 2 as
  * published by the Free Software Foundation.
  */
-
-#define DEBUG
-
 #include <linux/clk.h>
 #include <linux/device.h>
 #include <linux/module.h>
